README.md

```diff
@@ -1 +1,185 @@
-# promoted-ruby-client
\ No newline at end of file
+# promoted-ruby-client
+
+Ruby client designed for calling Promoted's Delivery and Metrics API.
+
+This version of the library only supports preparing objects for logging.  TODO - support Delivery API.
+
+## Expected pseudo-code flow for Metrics logging
+
+This example is for the integration where we do not want to modify the list of items to include `insertionId`.  TODO - add this example too.
+
+```
+def get_items args
+  items = retrieve_items((args))
+  async_log_request(items)
+  return items
+end
+
+# Done async
+def log_request items
+  log_request = Promoted::Ruby::Client.prepare_for_logging(input)
+  # Send JSON to Event API.
+  log_to_promoted_event_api(log_request)
+end
+```
+
+## Example to run the client
+
+1. Clone the repo on your local machine
+2. `cd promoted-ruby-client`
+3. `bundle`
+4. `irb -Ilib -rpromoted/ruby/client`
```

Comment:
**Note**: These instructions were for us to test it. After we publish this. We have to update these.

Reply:
Makes sense.  The README should have the external customer use.  Do you want these in a different markdown file for development?